src/app/status/page.tsx

```diff
@@ -17,7 +17,6 @@ const NodeAnimation = dynamic(
 )
 
 const StatusPage = async () => {
-  const initialData = await getNodeData().catch(() => undefined)
```

Comment:
https://vercel.com/docs/infrastructure/data-cache/manage-data-cache#disabling-vercel-data-cache
This removes the vercel forced cache for first request. 
```
export const fetchCache = 'force-no-store';
```
add it somewhere on this page file